pkg/signing/signing.go

```diff
@@ -281,14 +281,12 @@ func Verify(cpath, url string, id uint32, isGroup bool, authToken string, noProm
 	// the selected data object is hashed for comparison against signature block's
 	sifhash := computeHashStr(&fimg, descr)
 
-	// load the public keys available locally from the cache
-	elist, err := sypgp.LoadPubKeyring()
-	if err != nil {
-		return fmt.Errorf("could not load public keyring: %s", err)
-	}
+	var authok string
```

Comment:
typo: authok -> author

Reply:
I'm not sure thats a typo... but I'll change it anyway.

Reply:
Westley is right, authok is meant to be read as "authorization OK". Maybe "authOk" or "authOK" to avoid confusion?

Reply:
`authOk` sounds good! I'll change it!

Reply:
Nope, we are both wrong and @devprincess is right, it's author.

Reply:
Okay, I'll hold off :+1: